src/ufoLib2/objects/glyph.py

```diff
@@ -208,55 +226,55 @@ def clear(self):
         del self._guidelines[:]
         self.image.clear()
 
-    def clearAnchors(self):
+    def clearAnchors(self) -> None:
         """Clears out anchors."""
         del self._anchors[:]
 
-    def clearContours(self):
+    def clearContours(self) -> None:
         """Clears out contours."""
         del self.contours[:]
 
-    def clearComponents(self):
+    def clearComponents(self) -> None:
         """Clears out components."""
         del self.components[:]
 
-    def clearGuidelines(self):
+    def clearGuidelines(self) -> None:
         """Clears out guidelines."""
         del self._guidelines[:]
 
-    def removeComponent(self, component):
+    def removeComponent(self, component: Component) -> None:
         """Removes :class:`.Component` object from the glyph's list of components."""
         self.components.remove(component)
 
-    def appendAnchor(self, anchor):
+    def appendAnchor(self, anchor: Union[Anchor, Mapping[str, Any]]) -> None:
         """Appends :class:`.Anchor` object to glyph's list of anchors."""
         if not isinstance(anchor, Anchor):
```

Comment:
This one doesn't have the same error message when the provided argument is not a `Mapping`. Should the message be everywhere, or should there be no message because there are types now?

Reply:
Sorry I'm talking about the error message that you added to `Font.appendGuideline` https://github.com/fonttools/ufoLib2/pull/73/files#diff-7e8f667fefb48a9213fd432500aaebfaR437

Reply:
Hrm yeah. I need to check all the defcon methods that take Mappings again.

Reply:
So actually, Font.appendGuideline, Glyph.appendGuideline and Glyph.appendAnchor will take a mapping, but Glyph.appendContour demands a Contour object.